ScriptHandler::clearCache : error during installation process:

## Issue details

I am trying to install wallabag on a centos 8 with postgresql 10 and php 7.3 and I have error durring the make install process (see log below). I have a running wallabag on debian/mariadb so I tried with mariadb instead of postgresql but I have the same error.

### Environment

* wallabag version 2.3.8
* How did you install wallabag? Via git clone then make install
* php version: 7.3
* OS: centos 8
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): MariaDB and postgresql 10

### Steps to reproduce/test case
> git clone
> make install
> then the process is stuck at this point:

```
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::buildBootstrap
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::clearCache
                                                                        
  [Symfony\Component\Debug\Exception\ContextErrorException]                    
  Warning: file_put_contents(/var/www/wallabag2/var/cache//usr/share/Modules/init/profile.sh/app/usr/share/Modules/init/profile.shDebugProjectContainerDeprecations.log): failed to open stream: No such file or directory           
                                                                               
  [Symfony\Component\Config\Exception\FileLocatorFileNotFoundException]        
  The file "/var/www/wallabag2/app/config/config_/usr/share/Modules/init/profile.sh.yml" does not exist.

Script Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::clearCache handling the post-cmd event terminated with an exception
                                                                                   
  [RuntimeException]                                                               
  An error occurred when executing the "'cache:clear --no-warmup'" command:        
                                                                                   
    [Symfony\Component\Debug\Exception\ContextErrorException]                      
    Warning: file_put_contents(/var/www/wallabag2/var/cache//usr/share/Modules/init/profile.sh/app/usr/share/Modules/init/profile.shDebugProjectContainerDeprecations.log): failed to open stream: No such file or directory             
                                                                                                                                                                      
    [Symfony\Component\Config\Exception\FileLocatorFileNotFoundException]          
    The file "/var/www/wallabag2/app/config/config_/usr/share/Modules/init/profile.sh.yml" does not exist.                                                    

install [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--no-custom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--ignore-platform-reqs] [--] [<packages>]...

  [Symfony\Component\Debug\Exception\ContextErrorException]                                                                                                  
  Warning: file_put_contents(/var/www/wallabag2/var/cache//usr/share/Modules/init/profile.sh/app/usr/share/Modules/init/profile.shDebugProjectContainerDeprecations.log): failed to open stream: No such file or directory                                                                                            
                                                                                                                                                                                                                                                                       
  [Symfony\Component\Config\Exception\FileLocatorFileNotFoundException]                                   
  The file "/var/www/wallabag2/app/config/config_/usr/share/Modules/init/profile.sh.yml" does not exist.  

make: *** [GNUmakefile:14: install] Error 1
```

Here it is

```
[root@centos www]# git clone https://github.com/wallabag/wallabag.git
Cloning into 'wallabag'...
remote: Enumerating objects: 73, done.
remote: Counting objects: 100% (73/73), done.
remote: Compressing objects: 100% (67/67), done.
remote: Total 51890 (delta 31), reused 16 (delta 4), pack-reused 51817
Receiving objects: 100% (51890/51890), 62.90 MiB | 6.44 MiB/s, done.
Resolving deltas: 100% (30138/30138), done.
[root@centos www]# chown -R filovitch:filovitch wallabag/
[root@centos www]# cd wallabag/
[root@centos wallabag]# su - filovitch
Last login: Mon Jan 13 09:53:20 CET 2020 on pts/0
[filovitch@centos ~]$ cd /var/www/wallabag/
[filovitch@centos wallabag]$ make install
[filovitch@centos wallabag]$ make install
composer.phar not found, we'll see if composer is installed globally.
Note: checking out '2.3.8'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 9bbafdaa Release wallabag 2.3.8
Loading composer repositories with package information
Installing dependencies from lock file
Package operations: 119 installs, 0 updates, 0 removals
  - Installing symfony/polyfill-ctype (v1.11.0): Downloading (100%)
  - Installing twig/twig (v1.40.1): Downloading (100%)
  - Installing paragonie/random_compat (v2.0.18): Downloading (100%)
  - Installing symfony/polyfill-php70 (v1.11.0): Downloading (100%)
  - Installing symfony/polyfill-util (v1.11.0): Downloading (100%)
  - Installing symfony/polyfill-php56 (v1.11.0): Downloading (100%)
  - Installing symfony/polyfill-mbstring (v1.11.0): Downloading (100%)
  - Installing symfony/symfony (v3.3.18): Downloading (100%)
  - Installing symfony/polyfill-intl-icu (v1.11.0): Downloading (100%)
  - Installing symfony/polyfill-apcu (v1.11.0): Downloading (100%)
  - Installing psr/simple-cache (1.0.1): Downloading (100%)
  - Installing psr/log (1.1.0): Downloading (100%)
  - Installing psr/link (1.0.0): Downloading (100%)
  - Installing psr/container (1.0.0): Downloading (100%)
  - Installing psr/cache (1.0.1): Downloading (100%)
  - Installing fig/link-util (1.0.0): Downloading (100%)
  - Installing doctrine/lexer (v1.0.1): Downloading (100%)
  - Installing doctrine/inflector (v1.1.0): Downloading (100%)
  - Installing doctrine/collections (v1.4.0): Downloading (100%)
  - Installing doctrine/cache (v1.6.2): Downloading (100%)
  - Installing doctrine/annotations (v1.4.0): Downloading (100%)
  - Installing doctrine/common (v2.7.3): Downloading (100%)
  - Installing react/promise (v2.7.1): Downloading (100%)
  - Installing guzzlehttp/streams (3.0.0): Downloading (100%)
  - Installing guzzlehttp/ringphp (1.1.1): Downloading (100%)
  - Installing guzzlehttp/guzzle (5.3.3): Downloading (100%)
  - Installing bdunogier/guzzle-site-authenticator (1.0.0): Downloading (100%)
  - Installing behat/transliterator (v1.2.0): Downloading (100%)
  - Installing jdorn/sql-formatter (v1.2.17): Downloading (100%)
  - Installing doctrine/doctrine-cache-bundle (1.3.5): Downloading (100%)
  - Installing doctrine/dbal (v2.5.13): Downloading (100%)
  - Installing doctrine/doctrine-bundle (1.10.3): Downloading (100%)
  - Installing craue/config-bundle (2.1.0): Downloading (100%)
  - Installing defuse/php-encryption (v2.2.1): Downloading (100%)
  - Installing zendframework/zend-eventmanager (3.2.1): Downloading (100%)
  - Installing zendframework/zend-code (2.6.3): Downloading (100%)
  - Installing ocramius/proxy-manager (1.0.2): Downloading (100%)
  - Installing doctrine/migrations (v1.5.0): Downloading (100%)
  - Installing doctrine/doctrine-migrations-bundle (v1.3.2): Downloading (100%)
  - Installing willdurand/jsonp-callback-validator (v1.1.0): Downloading (100%)
  - Installing friendsofsymfony/jsrouting-bundle (1.6.3): Downloading (100%)
  - Installing friendsofsymfony/oauth2-php (1.2.3): Downloading (100%)
  - Installing friendsofsymfony/oauth-server-bundle (1.6.2): Downloading (100%)
  - Installing willdurand/negotiation (v2.3.1): Downloading (100%)
  - Installing friendsofsymfony/rest-bundle (2.4.0): Downloading (100%)
  - Installing friendsofsymfony/user-bundle (v2.0.2): Downloading (100%)
  - Installing grandt/relativepath (1.0.2): Downloading (100%)
  - Installing grandt/binstring (1.0.0): Downloading (100%)
  - Installing grandt/phpzipmerge (1.0.4): Downloading (100%)
  - Installing phpzip/phpzip (2.0.8): Downloading (100%)
  - Installing grandt/phpresizegif (1.0.3): Downloading (100%)
  - Installing grandt/phpepub (4.0.7.1): Downloading (100%)
  - Installing hoa/exception (1.17.01.16): Downloading (100%)
  - Installing hoa/consistency (1.17.05.02): Downloading (100%)
  - Installing hoa/event (1.17.01.13): Downloading (100%)
  - Installing hoa/iterator (2.17.01.10): Downloading (100%)
  - Installing hoa/visitor (2.17.01.16): Downloading (100%)
  - Installing hoa/ustring (4.17.01.16): Downloading (100%)
  - Installing hoa/protocol (1.17.01.14): Downloading (100%)
  - Installing hoa/zformat (1.17.01.10): Downloading (100%)
  - Installing hoa/regex (1.17.01.13): Downloading (100%)
  - Installing hoa/math (1.17.05.16): Downloading (100%)
  - Installing hoa/stream (1.17.02.21): Downloading (100%)
  - Installing hoa/file (1.17.07.11): Downloading (100%)
  - Installing hoa/compiler (3.17.08.08): Downloading (100%)
  - Installing hoa/ruler (2.17.05.16): Downloading (100%)
  - Installing html2text/html2text (4.2.1): Downloading (100%)
  - Installing incenteev/composer-parameter-handler (v2.1.3): Downloading (100%)
  - Installing wallabag/tcpdf (6.2.26): Downloading (100%)
  - Installing true/punycode (v2.1.1): Downloading (100%)
  - Installing smalot/pdfparser (v0.14.0): Downloading (100%)
  - Installing simplepie/simplepie (1.5.2): Downloading (100%)
  - Installing monolog/monolog (1.24.0): Downloading (100%)
  - Installing j0k3r/safecurl (2.0.4): Downloading (100%)
  - Installing electrolinux/php-html5lib (0.1.0): Downloading (100%)
  - Installing j0k3r/php-readability (1.2.3): Downloading (100%)
  - Installing j0k3r/graby-site-config (1.0.83): Downloading (100%)
  - Installing fossar/htmlawed (1.2.7): Downloading (100%)
  - Installing j0k3r/graby (1.20.0): Downloading (100%)
  - Installing javibravo/simpleue (2.1.0): Downloading (100%)
  - Installing phpoption/phpoption (1.5.0): Downloading (100%)
  - Installing phpcollection/phpcollection (0.5.0): Downloading (100%)
  - Installing jms/parser-lib (1.0.0): Downloading (100%)
  - Installing jms/metadata (1.7.0): Downloading (100%)
  - Installing doctrine/instantiator (1.0.5): Downloading (100%)
  - Installing jms/serializer (1.14.0): Downloading (100%)
  - Installing kphoen/rulerz (0.20.5): Downloading (100%)
  - Installing kphoen/rulerz-bridge (1.0.1): Downloading (100%)
  - Installing kphoen/rulerz-bundle (0.14.1): Downloading (100%)
  - Installing doctrine/orm (v2.5.14): Downloading (100%)
  - Installing lexik/form-filter-bundle (v5.0.10): Downloading (100%)
  - Installing liip/theme-bundle (1.6.1): Downloading (100%)
  - Installing mgargano/simplehtmldom (1.5): Downloading (100%)
  - Installing mnapoli/piwik-twig-extension (1.0.0): Downloading (100%)
  - Installing michelf/php-markdown (1.8.0): Downloading (100%)
  - Installing nelmio/api-doc-bundle (2.13.3): Downloading (100%)
  - Installing nelmio/cors-bundle (1.5.5): Downloading (100%)
  - Installing php-amqplib/php-amqplib (v2.9.2): Downloading (100%)
  - Installing php-amqplib/rabbitmq-bundle (v1.14.4): Downloading (100%)
  - Installing predis/predis (v1.1.1): Downloading (100%)
  - Installing sonata-project/google-authenticator (1.1.0): Downloading (100%)
  - Installing scheb/two-factor-bundle (v2.14.0): Downloading (100%)
  - Installing composer/ca-bundle (1.1.4): Downloading (100%)
  - Installing sensiolabs/security-checker (v5.0.3): Downloading (100%)
  - Installing sensio/distribution-bundle (v5.0.24): Downloading (100%)
  - Installing sensio/framework-extra-bundle (v3.0.29): Downloading (100%)
  - Installing gedmo/doctrine-extensions (v2.4.37): Downloading (100%)
  - Installing stof/doctrine-extensions-bundle (v1.3.0): Downloading (100%)
  - Installing sulu/symfony-intl-fix (1.0.0): Downloading (100%)
  - Installing symfony/monolog-bundle (v3.2.0): Downloading (100%)
  - Installing swiftmailer/swiftmailer (v5.4.12): Downloading (100%)
  - Installing symfony/swiftmailer-bundle (v2.6.7): Downloading (100%)
  - Installing twig/extensions (v1.5.4): Downloading (100%)
  - Installing wallabag/php-mobi (1.0.1): Downloading (100%)
  - Installing pagerfanta/pagerfanta (v1.1.0): Downloading (100%)
  - Installing white-october/pagerfanta-bundle (v1.2.4): Downloading (100%)
  - Installing willdurand/hateoas (2.12.0): Downloading (100%)
  - Installing jms/serializer-bundle (2.4.4): Downloading (100%)
  - Installing willdurand/hateoas-bundle (1.4.0): Downloading (100%)
Generating optimized autoload files
> Incenteev\ParameterHandler\ScriptHandler::buildParameters
Creating the "app/config/parameters.yml" file
Some parameters are missing. Please provide them.
database_driver (pdo_mysql):
database_driver_class (null):
database_host (127.0.0.1):
database_port (null):
database_name (wallabag):
database_user (root): wallabag
database_password (null): wallabagpwd
database_path (null):
database_table_prefix (wallabag_):
database_socket (null):
database_charset (utf8mb4):
domain_name ('https://your-wallabag-url-instance.com'): 'http://wallabag.test.be'
mailer_transport (smtp):
mailer_host (127.0.0.1):
mailer_user (null):
mailer_password (null):
locale (en):
secret (ovmpmAWXRCabNlMgzlzFXDYmCFfzGv):
twofactor_auth (true): false
twofactor_sender (no-reply@wallabag.org):
fosuser_registration (true): false
fosuser_confirmation (true): false
from_email (no-reply@wallabag.org):
rss_limit (50):
rabbitmq_host (localhost):
rabbitmq_port (5672):
rabbitmq_user (guest):
rabbitmq_password (guest):
rabbitmq_prefetch_count (10):
redis_scheme (tcp):
redis_host (localhost):
redis_port (6379):
redis_path (null):
redis_password (null):
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::buildBootstrap
> Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::clearCache


  [Symfony\Component\Debug\Exception\ContextErrorException]
  Warning: file_put_contents(/var/www/wallabag/var/cache//usr/share/Modules/i
  nit/profile.sh/app/usr/share/Modules/init/profile.shDebugProjectContainerDe
  precations.log): failed to open stream: No such file or directory



  [Symfony\Component\Config\Exception\FileLocatorFileNotFoundException]
  The file "/var/www/wallabag/app/config/config_/usr/share/Modules/init/profi
  le.sh.yml" does not exist.


Script Sensio\Bundle\DistributionBundle\Composer\ScriptHandler::clearCache handling the post-cmd event terminated with an exception


  [RuntimeException]
  An error occurred when executing the "'cache:clear --no-warmup'" command:





    [Symfony\Component\Debug\Exception\ContextErrorException]
    Warning: file_put_contents(/var/www/wallabag/var/cache//usr/share/Modules/i
    nit/profile.sh/app/usr/share/Modules/init/profile.shDebugProjectContainerDe
    precations.log): failed to open stream: No such file or directory



    [Symfony\Component\Config\Exception\FileLocatorFileNotFoundException]
    The file "/var/www/wallabag/app/config/config_/usr/share/Modules/init/profi
    le.sh.yml" does not exist.


install [--prefer-source] [--prefer-dist] [--dry-run] [--dev] [--no-dev] [--no-custom-installers] [--no-autoloader] [--no-scripts] [--no-progress] [--no-suggest] [-v|vv|vvv|--verbose] [-o|--optimize-autoloader] [-a|--classmap-authoritative] [--apcu-autoloader] [--ignore-platform-reqs] [--] [<packages>]...



  [Symfony\Component\Debug\Exception\ContextErrorException]
  Warning: file_put_contents(/var/www/wallabag/var/cache//usr/share/Modules/init/profile.sh/app/usr/share/Modules/init/profile.shDebugProjectContainerDeprecations.log): failed to open stream: No such file or directory



  [Symfony\Component\Config\Exception\FileLocatorFileNotFoundException]
  The file "/var/www/wallabag/app/config/config_/usr/share/Modules/init/profile.sh.yml" does not exist.


make: *** [GNUmakefile:14: install] Error 1
[filovitch@centos wallabag]$
```

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -3,12 +3,9 @@ TMP_FOLDER=/tmp
 RELEASE_FOLDER=wllbg-release
 
 # ensure the ENV variable is well defined
-ifeq ($(origin ENV), prod)
-	# all good ("prod" is a valid env)
-else ifeq ($(origin ENV), dev)
-	# all good ("dev" is a valid env)
-else ifeq ($(origin ENV), test)
-	# all good ("test" is a valid env)
+AVAILABLE_ENV := prod dev test
+ifneq ($(filter $(ENV),$(AVAILABLE_ENV)),)
+	# all good
 else
 	# not good, force it to "prod"
 	override ENV = prod

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -2,7 +2,17 @@ SHELL=bash
 TMP_FOLDER=/tmp
 RELEASE_FOLDER=wllbg-release
 
-ENV ?= prod
+# ensure the ENV variable is well defined
+ifeq ($(origin ENV), prod)
+	# all good ("prod" is a valid env)
+else ifeq ($(origin ENV), dev)
+	# all good ("dev" is a valid env)
+else ifeq ($(origin ENV), test)
+	# all good ("test" is a valid env)
+else
+	# not good, force it to "prod"
+	override ENV = prod
+endif
 
 help: ## Display this help menu
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'

